packages/elements/src/components/ino-carousel/ino-carousel.tsx

```diff
@@ -0,0 +1,168 @@
+import { Component, ComponentInterface, Element, Host, Prop, Watch, h } from '@stencil/core';
+import classNames from 'classnames';
+
+
+@Component({
+  tag: 'ino-carousel',
+  styleUrl: 'ino-carousel.scss',
+  shadow: false
+})
+export class InoCarousel implements ComponentInterface {
+
+  @Element() el: HTMLElement;
+  private slides: HTMLInoCarouselSlideElement[];
+  private currentSlide: number = 0;
+  private slideCounter: number = 1;
+  private timer: NodeJS.Timeout;
+
+  /**
+   * Optional group value to manually manage the displayed slide
+   */
+  @Prop() value?: any;
+
+  @Watch('value')
+  valueChanged(newVal: any) {
+    this.addAnimationToSlide(this.slides[this.currentSlide]); // adds the slide animation to the current slide
+    this.selectSlide(newVal);
+    this.addAnimationToSlide(this.slides[this.currentSlide]); // adds the slide animation to the new slide
+  }
+
+  /**
+   * Enables autoplay which causes slides to be changed automatically
+   */
+  @Prop() inoAutoplay: boolean = false;
+  @Watch('inoAutoplay')
+  inoAutoplayChanged() {
+    this.configureAutoplay();
+  }
+
+  /**
+   * Enables the slide animation
+   */
+  @Prop() inoAnimated: boolean = false;
+
+  /**
+   * Hides the arrow buttons
+   */
+  @Prop() inoHideButtons: boolean = false;
+
+  /**
+   * Restarts playback from the first slide upon reaching the last slide
+   */
+  @Prop() inoInfinite: boolean = false;
+
+  /**
+   * Sets the intermission between two slides (Unit: ms)
+   */
+  @Prop() inoIntermission: number = 5000;
+
+  /**
+   * Enables reverse playback of the slides
+   */
+  @Prop() inoReverse: boolean = false;
+
+  componentDidLoad(): void {
+    this.slides = this.getSlides();
+    this.configureSlides();
+    this.configureAutoplay();
+  }
+
+  /**
+   * adds a slide animation to the slide with the given index
```

Comment:
This comment seems to be outdated. The function does not work with indices anymore

Reply:
✅